exporter/opentelemetry-exporter-prometheus-remote-write/README.rst

```diff
@@ -0,0 +1,23 @@
+OpenTelemetry Prometheus Remote Write Exporter
+==============================================
+
+This library allows exporting metric data to `Prometheus Write Integrated Backends
```

Comment:
Should this say Prometheus Remote Write?

Reply:
Updated